app/assets/javascripts/map/presenters/tabs/SubscriptionPresenter.js

```diff
@@ -0,0 +1,500 @@
+/**
+ * The SubscriptionPresenter class for the AnalysisToolView.
+ *
+ * @return SubscriptionPresenter class.
+ */
+define([
+  'map/presenters/PresenterClass',
+  'underscore',
+  'backbone',
+  'mps',
+  'topojson',
+  'helpers/geojsonUtilsHelper',
+  'map/services/CountryService',
+  'map/services/RegionService'
+], function(PresenterClass, _, Backbone, mps, topojson, geojsonUtilsHelper, countryService, regionService) {
+
+  'use strict';
+
+  var StatusModel = Backbone.Model.extend({
+    defaults: {
+      baselayer: null,
+      both: false,
+      resource: null, // analysis resource
+      date: null,
+      threshold: 30, // by default
+      iso: null,
+      overlay: null, // google.maps.Polygon (user drawn polygon)
+      multipolygon: null, // geojson (countries and regions multypolygon)
+      disableUpdating: false,
+      dont_analyze: false
+    }
+  });
+
+  var concessionsSql = {
```

Comment:
I know it's just in the purpose of beauty but maybe we should use some URL templating here, what do you think? @adammulligan